### PR TITLE
feat(DDL)

### PR DESCRIPTION
**Pull request by stavbi1**

### CODE REVIEW

**Comment by adirsiton** [comment]
Next step is to create a db insance over openshift?

File: scripts/DDL.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	standard_id int REFERENCES standards(id) ON DELETE CASCADE,
+	entry_date Date,
+	seniority int,
+	is_damach boolean,
+	advertiser_upn text REFERENCES users(upn) ON DELETE CASCADE,
+	contact text,
+	base_location text
+);
+
+CREATE TABLE tags_of_adds(
```
Comment (michael-hoffman-26) [inline scripts/DDL.sql]: what is the meaning of tag?

Comment (JetoPistola) [reply scripts/DDL.sql]: https://user-images.githubusercontent.com/62653606/77565962-a8f1ee80-6ecd-11ea-9bda-45c4ee526643.png

Look at this 😄

Comment (michael-hoffman-26) [reply scripts/DDL.sql]: Ok I got it for every addvisiter you can add few tags, Stav told me it will be used for thh search

**Pull request merged**